src/main/java/com/solvd/carina/demo/gui/demoblaze/NavBar.java

```diff
@@ -0,0 +1,59 @@
+package com.solvd.carina.demo.gui.demoblaze;
+
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.gui.AbstractUIObject;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+public class NavBar extends AbstractUIObject {
+
+    @FindBy(xpath = "//*[@id=\"navbarExample\"]/ul/li[1]/a")
```

Comment:
please try to avoid such locators with only tags, try to explore dom tree and find some another attributes(id, text e.t.c)